packages/ffe-context-message-react/src/ContextTipMessage.js

```diff
@@ -3,7 +3,7 @@ import ContextMessage from './ContextMessage';
 import { LyspareIkon } from '@sb1/ffe-icons-react';
 
 const ContextTipMessage = props => (
-    <ContextMessage {...props} messageType="tip" icon={<LyspareIkon />} />
+    <ContextMessage {...props} messageType="tips" icon={<LyspareIkon />} />
 );
 
 export default ContextTipMessage;
```

Comment:
Burde da ikke klassen også hete `ContextTipsMessage`?

Reply:
Du tenker på komponenten? Jeg tenkte det samme, men så var jeg litt redd for at det ble en litt for stor endring, men det er vel ikke noen større endring enn den jeg allerede gjør sånn sett 😅

Reply:
Endringen er jo allerede en breaking change. Da kan man også ordne opp i dette, synes jeg.